ops/deployer/gauntlet/gauntlet.go

```diff
@@ -0,0 +1,526 @@
+		"link",
+		"ocr2",
+	)
+
+	if err != nil {
+		return errors.New("Billing AC initialization failed")
+	}
+	return msg.Check(nil)
+}
+
+func (t *GauntlerDeployer) DeployLINK() error {
+	fmt.Println("Deploying LINK Token...")
+	err := t.gauntlet.ExecCommand(
+		"token:deploy",
+		t.gauntlet.Flag("network", t.network),
+	)
+	if err != nil {
+		return errors.New("LINK contract deployment failed")
+	}
+
+	report, err := t.gauntlet.ReadCommandReport()
+	fmt.Println(report)
+	if err != nil {
+		fmt.Println(err)
+		return errors.New("report not available")
+	}
+
+	linkAddress := report.Responses[0].Contract
+	t.Account[LINK] = linkAddress
+
+	msg := utils.LogStatus("Deployed LINK token")
+
+	return msg.Check(nil)
+}
+
+type OCRinit struct {
+	LinkToken                 string `json:"linkToken"`
+	MinAnswer                 string `json:"minAnswer"`
+	MaxAnswer                 string `json:"maxAnswer"`
+	BillingAccessController   string `json:"billingAccessController"`
+	RequesterAccessController string `json:"requesterAccessController"`
+	Decimals                  int    `json:"decimals"`
+	Description               string `json:"description"`
+}
+
+func (t *GauntlerDeployer) DeployOCR() error {
+	msg := utils.LogStatus("Deployed OCR contract")
+
+	fmt.Println("Deploying OCR Feed:")
+	fmt.Println("Step 1: Init Requester Access Controller")
+	err := t.gauntlet.ExecCommand(
+		"access_controller:deploy",
+		t.gauntlet.Flag("network", t.network),
+	)
+	if err != nil {
+		return errors.New("Request AC initialization failed")
+	}
+	report, err := t.gauntlet.ReadCommandReport()
+	if err != nil {
+		return err
+	}
+	t.Account[RequesterAccessController] = report.Responses[0].Contract
+
+	fmt.Println("Step 2: Init Billing Access Controller")
+	err = t.gauntlet.ExecCommand(
+		"access_controller:deploy",
+		t.gauntlet.Flag("network", t.network),
+	)
+	if err != nil {
+		return errors.New("Billing AC initialization failed")
+	}
+	report, err = t.gauntlet.ReadCommandReport()
+	if err != nil {
+		return err
+	}
+	t.Account[BillingAccessController] = report.Responses[0].Contract
+
+	fmt.Println("Step 6: Init OCR 2 Feed")
+	input := OCRinit{
+		MinAnswer:                 "0",
+		MaxAnswer:                 "10000000000",
+		Decimals:                  2,
+		Description:               "Hello",
+		RequesterAccessController: t.Account[RequesterAccessController],
+		BillingAccessController:   t.Account[BillingAccessController],
+		LinkToken:                 t.Account[LINK],
+	}
+
+	jsonInput, err := json.Marshal(input)
+	if err != nil {
+		return err
+	}
+
+	err = t.gauntlet.ExecCommand(
+		"ocr2:deploy",
+		t.gauntlet.Flag("network", "bombay-testnet"),
+		t.gauntlet.Flag("input", string(jsonInput)),
+	)
+	if err != nil {
+		return errors.New("feed initialization failed")
+	}
+
+	report, err = t.gauntlet.ReadCommandReport()
+	if err != nil {
+		return err
+	}
+
+	t.Account[OCR2] = report.Responses[0].Contract
+	fmt.Printf(" - %s", report.Data["state"])
```

Comment:
Not sure if this report contains a `state`, that seems more for Solana

Reply:
it does seem to print one out. It prints a green check emoji or a red X